src/DSharp.Mscorlib/Collections/Dictionary.cs

```diff
@@ -1,18 +1,22 @@
 ﻿using System.Runtime.CompilerServices;
-
+// System.Collections.Dictionary doesn't exist in CLR. It does exist:
+// System.Collections.IDictionary
+// System.Collections.DictionaryBase
 namespace System.Collections
 {
     [ScriptIgnoreNamespace]
     [ScriptImport]
     [ScriptName("Object")]
-    public sealed class Dictionary : IEnumerable
+    // In CLR this is an interface and not a class, therefore no ctors are defined there
+    public sealed partial class Dictionary : IEnumerable //In CLR this gui inherits from ICollection that inherits from IEnumerable
```

Comment:
Maybe we can mark this for deprecation to be replaced with JSDictionary (Which would be new). The idea being this would map to a JSObject, which is also a dictionary :)

Reply:
`Dictionary` was removed and `Dictionary<,>` was completely implemented, we should be able to swap these with the generic one or `IDictionary`